.github/workflows/swift-toolchain.yml

```diff
@@ -12,13 +18,11 @@ on:
       swift_tag:
         description: 'Swift Build Tag'
         required: false
-
```

Comment:
Do we need these whitespace changes (I think empty lines are removed?)?

Reply:
Ah, this is just me regretting that we don't have a YAML formatter.  I removed these spaces because in my last diff it looked like I was the one that added them, but I was mistaken.

Reply:
Fixed!